src/annotation.js

```diff
@@ -123,7 +123,16 @@ module.exports = function(Chart) {
 			}
 		},
 		destroy: function(chartInstance) {
-			var deregisterers = chartInstance.annotation.onDestroy;
+			var deregisterers;
+			if (chartInstance) {
```

Comment:
It would be cleaner to write this as:
```
if (!chartInstance || !chartInstance.annotation) {
  return;
}
deregisterers = chartInstance.annotation.onDestroy;
```